examples/decoding/plot_haxby_anova_svm.py

```diff
@@ -46,7 +46,7 @@
 
 ### Prediction function #######################################################
 
-### Define the prediction function to be used.
+# Define the prediction function to be used.
```

Comment:
OK, I think that the leading `###` are not here to indicate a level of comment but just for cosmetics. In that cas, removing them is fine for me.